### DISCUSSION TITLE
Azure AD B2B/B2C konsultant

### DISCUSSION THREAD

**Discussion by arczibdg** [AzureCommunityPL/group#337] [Oferty pracy]
Cześć,

Poszukujemy konsultanta na 3 tygodnie od początku września, która pomoże w tematach związanych z Azure AD/B2B/B2C.
Praca w pełni zdalna. Stawka 200PLN/h do lekkiej negocjacji.

Chętnych proszę o kontakt na hr@incredevo.com.

Pozdrawiam,
Artur

**Comment by tonyszko** [comment]
Jakby co to daj znać na priv albo firmowy - tomasz.onyszko@predicagroup.com - w AAD B2C potrafimy i to mocno :)
